packages/localize-tools/src/tests/analysis.unit.test.ts

```diff
@@ -342,7 +342,28 @@ test('error: different message contents', () => {
       },
     ],
     [
-      '__DUMMY__.ts(4,5): error TS2324: Message ids must have the same default text wherever they are used',
+      '__DUMMY__.ts(3,5): error TS2324: The translation message with ID greeting was defined in 2 places, but with different strings or descriptions. If these messages should be translated together, make sure their strings and descriptions are identical, and consider factoring out a common variable or function. If they should be translated separately, add one or more {id: "..."} overrides to distinguish them.',
+    ]
+  );
+});
+
+test('error: same message contents with different desc', () => {
```

Comment:
Mind adding one more test case for `same id and desc with different expression is not error`?